amp-homework/src/app/app.module.ts

```diff
@@ -10,6 +10,8 @@ import { CoursesComponent } from './pages/courses/courses/courses.component';
 import { CourseDetailsComponent } from './pages/courses/course-details/course-details.component';
 import { LoginComponent } from './pages/login/login/login.component';
 import { ToolBoxComponent } from './pages/courses/tool-box/tool-box.component';
+import { CoursesService } from './pages/courses/courses.service';
+import { AuthService } from '../app/common/auth.service';
```

Comment:
No need to point upper level like 'app'

import { AuthService } from './common/auth.service';